tests/test_jetpack.py

```diff
@@ -62,111 +69,94 @@ def test_bad_harnessoptions():
-    err = _do_test(MockXPI({"bootstrap.js": bootstrap,
+    err = _do_test(MockXPI({"bootstrap.js": bootstrap_jpm,
                             "package.json": "{}"}))
-    print err.print_summary(verbose=True)
+
     assert not err.failed()
-    assert "is_jetpack" in err.metadata and err.metadata["is_jetpack"]
+    assert err.metadata.get("is_jetpack")
 
-    # Test that all files are marked as pretested.
-    pretested_files = err.get_resource("pretested_files")
-    assert pretested_files
-    assert "bootstrap.js" in pretested_files
+    assert_framework_files(err, ["bootstrap.js"])
 
 
 def test_package_json_different_bootstrap():
     """Test that a minimalistic package.json Jetpack setup will pass."""
 
-    err = _do_test(MockXPI({"bootstrap.js": "var foo = 'bar';",
+    err = _do_test(MockXPI({"bootstrap.js": bootstrap_jpm,
                             "package.json": "{}"}))
-    print err.print_summary(verbose=True)
     assert not err.failed()
-    assert "is_jetpack" in err.metadata and err.metadata["is_jetpack"]
+    assert err.metadata.get("is_jetpack")
 
-    # Test that all files are not marked as pretested.
-    pretested_files = err.get_resource("pretested_files")
-    assert not pretested_files
-    assert "bootstrap.js" not in pretested_files
+    assert_framework_files(err, ["bootstrap.js"])
 
 
 def test_missing_elements():
     """Test that missing elements in harness-options will fail."""
 
-    harnessoptions = {"sdkVersion": "1.8-dev",
+    harnessoptions = {"sdkVersion": "1.7",
                       "jetpackID": ""}
 
-    with open("tests/resources/bootstrap.js") as bootstrap_file:
-        bootstrap = bootstrap_file.read()
-
     err = _do_test(MockXPI({"bootstrap.js": bootstrap,
-                            "harness-options.json":
-                                json.dumps(harnessoptions)}))
+                            "harness-options.json": harnessoptions}))
     assert err.failed()
 
 
 def test_skip_safe_files():
-    """Test that missing elements in harness-options will fail."""
+    """
+    Tests that errors aren't triggered by the presence of certain files.
+    I'm not actually certain what the purpose of this test is, but the previous
+    docstring was for a different test.
+    """
 
-    harnessoptions = {"sdkVersion": "1.8-dev",
+    harnessoptions = {"sdkVersion": "1.7",
                       "jetpackID": "",
                       "manifest": {}}
 
-    with open("tests/resources/bootstrap.js") as bootstrap_file:
-        bootstrap = bootstrap_file.read()
-
-    err = _do_test(MockXPI({"bootstrap.js": bootstrap,
-                            "harness-options.json":
-                                json.dumps(harnessoptions),
-                            "foo.png": True,
-                            "bar.JpG": True,
-                            "safe.GIF": True,
-                            "icon.ico": True,
-                            "foo/.DS_Store": True}))
```

Comment:
No. That file generates a warning, and there didn't seem to be a good reason to keep it.